source/features/conversation-links-on-repo-lists.tsx

```diff
@@ -44,7 +44,7 @@ function init(signal: AbortSignal): void {
 
 void features.add(import.meta.url, {
 	include: [
-		pageDetect.isProfileRepoList,
+		pageDetect.isUserProfileRepoTab,
```

Comment:
We need to add a comment too, mentioning that the org already has it

Reply:
[`a23f32c` (#7185)](https://github.com/refined-github/refined-github/pull/7185/commits/a23f32c77ed32a4d409c8dccd34eb00157277bb4)